src/Database/PostgreSQL/PQTypes/Internal/Connection.hs

```diff
@@ -125,6 +132,11 @@ newtype Connection = Connection
   { unConnection :: MVar (Maybe ConnectionData)
   }
 
+getBackendPidIO :: Connection -> IO Int
+getBackendPidIO conn = do
+  withConnectionData conn "getBackendPidIO" $ \cd -> do
+    pure (cd, cdBackendPid cd)
```

Comment:
I would dispense with the `do` here, to be honest, but I guess from our hlint.yaml there is no in-house policy against redundant `do` ?

Reply:
While `do` is technically redundant, it's there to appease haskell-mode formatter in emacs to do the right thing :ghost: